plugins/Essentials/config.yml

```diff
@@ -711,7 +711,7 @@ newbies:
   # If so, use this format, replacing {DISPLAYNAME} with the player name.
   # If not, set to ''
   #announce-format: ''
-  announce-format: '&dWelcome {DISPLAYNAME}&d to the server!'
+  announce-format: '&dWelcome {DISPLAYNAME}&d to SpigotCraft server!'
```

Comment:
this doesn't make sense grammatically, either 'to the server' or 'to the SpigotCraft server', not just 'to SpigotCraft server'